js/app/controllers/editorcontroller.js

```diff
@@ -53,15 +53,16 @@ app.controller('EditorController', ['$scope', 'TimezoneService', 'AutoCompletion
 			{title: t('calendar', 'Details'), value: 0},
 			{title: t('calendar', 'Attendees'), value: 1},
 			{title: t('calendar', 'Reminders'), value: 2},
-			{title: t('calendar', 'Repeating'), value: 3}
+			{title: t('calendar', 'Repeating'), value: 3},
+			{title: t('calendar', 'Spreed Meeting'), value: 4}
```

Comment:
Call Meeting?

At least not introduce a brand like name that is not used anywhere else in the user interface